src/themes/dspace/app/header-nav-wrapper/header-navbar-wrapper.component.scss

```diff
@@ -1,7 +1,91 @@
 :host {
-  // The header-navbar-wrapper should not have a z-index, otherwise it would cover the media viewer despite its higher z-index
-  position: relative;
+  --ds-header-navbar-border-bottom-style: solid var(--ds-header-navbar-border-bottom-height) var(--ds-header-navbar-border-bottom-color);
+  --ds-mobile-navbar-border-top-style: solid var(--ds-mobile-navbar-border-top-height) var(--ds-mobile-navbar-border-top-color);
+  --ds-collapsible-navbar-height: calc(100vh - var(--ds-header-height));
+
   div#header-navbar-wrapper {
-    border-bottom: 5px var(--ds-header-navbar-border-bottom-color) solid;
+    // The header-navbar-wrapper should not have a z-index, otherwise it would cover the media viewer despite its higher z-index
+    position: relative; // required by the mobile collapsible navbar
+    border-bottom: var(--ds-header-navbar-border-bottom-style); // gets covered by mobile navbar wrapper, when open
+
+    div#mobile-navbar-wrapper {
+      width: 100%;
+      background-color: var(--ds-expandable-navbar-bg);
+      position: absolute;
+      z-index: var(--ds-nav-z-index);
+      overflow: hidden;
+
+      nav#collapsible-mobile-navbar {
+        // Following parameters are changed by slideMobileNav animation
+        min-height: var(--ds-collapsible-navbar-height);
+        height: auto;
+        border-bottom: var(--ds-header-navbar-border-bottom-style);
+      }
+    }
+  }
+
+  /* MENU ITEMS */
+
+  ::ng-deep {
+    .ds-menu-item, .ds-menu-toggler-wrapper {
+      white-space: nowrap;
+      text-decoration: none;
+    }
+
+    nav#desktop-navbar ::ng-deep {
+
+      // desktop menu
+      .ds-menu-item-wrapper, .ds-menu-item, .ds-menu-toggler-wrapper {
+        // fill navbar height (required by dropdown menus) and center content
+        display: flex;
+        align-items: center;
+        height: 100%;
+      }
+      .ds-menu-item {
+        padding: 0.5rem;
```

Comment:
This should be (derived from) a variable. I also get the impression that dropdown menu options had more padding-left and right before this PR